### PR TITLE
allow space and dot in project filename

### PR DESCRIPTION
**Pull request by nworr**
project file name can now have space or . in it (test.cats.qgs, "my filename.qgs" 

Ticket : #2353   and #2020

Funded by 3Liz

### CODE REVIEW

**Comment by mdouchin** [comment]
@nworr I just added the tag "Run cypress" to see if it is green

**Comment by laurentj** [comment]
I don't believe it fixes #2020. Some unit tests should be added (probably by renaming a test project ?)

**Review by rldhont — CHANGES_REQUESTED** [review]
Adding tests

**Comment by nworr** [comment]
new test added : 
 - project file with space / dot : [85d9a00](https://github.com/3liz/lizmap-web-client/pull/3289/commits/85d9a0031ac1b8e75e8d9e86f17b9d03e5ab0242)
 - cypress test launch projet and check if layer exists  [9ee5777](https://github.com/3liz/lizmap-web-client/pull/3289/commits/9ee577732193a88c665306094bbc9e910e3a3065)

**Comment by nworr** [comment]
working on a  playwright test instead of cypress

File: tests/end2end/cypress/integration/projects_filename_dot_space-ghaction.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+describe('Filename with dot or space', () => {
+    
+    it('projet with dot or spacr can be loaded', () => {
```
Comment (Gustry) [inline tests/end2end/cypress/integration/projects_filename_dot_space-ghaction.js:3]:
```suggestion
    it('projet with dot or space can be loaded', () => {
```

**Review by Gustry — APPROVED** [review]
LGTM

**Review by rldhont — APPROVED** [review]

**Review by laurentj — APPROVED** [review]

**Pull request merged**